services/vision/mlvision/detector.go

```diff
@@ -110,7 +112,7 @@ func attemptToBuildDetector(mlm mlmodel.Service,
 			return nil, err
 		}
 
-		boundingBoxes, err := ml.FormatDetectionOutputs(outNameMap, outMap, origW, origH, boxOrder, labels)
+		boundingBoxes, err := ml.FormatDetectionOutputs(outNameMap, outMap, origW, origH, boxOrder, labels, wasResized, resizeW, resizeH)
```

Comment:
Since `resizeW` and `resizeH` could be set to the `origW` and `origH`, then just passing `resizeW` and `resizeH` in place of `origW` and `origH` in this method should fix the issue without any change to the method. 

```suggestion
		boundingBoxes, err := ml.FormatDetectionOutputs(outNameMap, outMap, resizeW, resizeH, boxOrder, labels)
```